packages/psss/src/containers/Tables/CountryReportTable.js

```diff
@@ -82,18 +82,25 @@ export const CountryReportTable = React.memo(
     const [totalSitesValue, setTotalSitesValue] = useState(totalSites);
     const { countryCode } = useParams();
 
-    const [saveReport] = useSaveCountryReport(countryCode);
+    useEffect(() => {
+      setSitesReportedValue(sitesReported);
+      setTotalSitesValue(totalSites);
+    }, [sitesReported, totalSites]);
+
+    const [saveReport] = useSaveCountryReport(countryCode, weekNumber);
 
     const handleSubmit = async () => {
       setTableStatus(TABLE_STATUSES.SAVING);
 
       try {
         await saveReport({
-          orgUnit: countryCode,
-          period: weekNumber,
-          ...fields,
-          sitesReported: parseInt(sitesReportedValue, 10),
-          totalSites: parseInt(totalSitesValue, 10),
+          afr: parseInt(fields['AFR-totalCases']),
```

Comment:
This is not the most beautiful piece of code but it is kind of necessary with the way the data is set up in the editable table. I welcome any suggestions for a more local fix or I could create a tech debt card to look at the editable table data keying. Or it might be ok since we are hardcoding the 5 sydrome ids anyway?